### ISSUE TITLE
[EN][Hunter][Talent] Longevity

### ISSUE THREAD

**Issue by smmuntz** [Atlantiss/BugTracker#3202]
Right now Longevity (http://www.wowwiki.com/Longevity) works properly for the most part, it reduces the cooldown  of Bestial Wrath, Intimidation and certain Pet Special Abilities, but not all of them. It seems to properly reduce the cooldown of the unique pet spells of each class (such as Roar of Courage, or Shell Shield, etc.) but does not reduce the cooldown of Cower or Growl. It visually reduces the cooldown of these abilities, they show that they have the reduced cooldown and their timers count down, but the spells are not actually available until their full cooldowns are reached after use.

I do know that Claw/Smack/Bite are not reduced by longevity, and that is proper, but Cower and Growl should be reduced. I had trouble finding any videos showing this, but here is a video showing reduced cooldown of Growl.

https://youtu.be/kSlJbEQ5dUI?t=3m30s

**Comment by Tj20** [comment]
That's true. When looking for how Longevity works exactly I found some comments which indicate that the CD of Growl should get reduced:

> Silverback: This fantastic talent is one of the keys that keeps your pet going – your pet heals 2% of it’s total health every time it growls. With the Longevity talent our hunter has deep in the BM tree, our pet growls even more often, healing itself even more often. (from [warcrafthuntersunion](http://www.warcrafthuntersunion.com/2009/07/extreme-solo-hunter-pet-build/))

> Don't forget about silverback. With longevity to put growl at 3.5s CD, and the 50% healing bonus from BotR and spirit bond, that's another 9% healing per 10.5 seconds. (from [elitistjerks](http://forums.elitistjerks.com/topic/96328-cataclysm-hunter-changes/page-62#entry1750550))

Now these are from pre-Cata, but I'm sure it wasn't changed in Cata. Unfortunately it's really hard to find comments about Longevity because apparently everybody just knew what it did back then and had no reason to write about it on the internet.

I think it's not the worst of our issues right now because Cower usually isn't used on CD and (unfortunately) Growl is bugged anyways (barely generates any aggro, #2931), but yes, I think you're right.
